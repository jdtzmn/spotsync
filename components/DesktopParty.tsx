import React, { Component } from 'react'

class DesktopParty extends Component {
  render() {
    return (
      <div />
    )
  }
}

export default DesktopParty
