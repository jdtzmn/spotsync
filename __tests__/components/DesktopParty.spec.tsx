import * as React from 'react'
import { shallow } from 'enzyme'
import DesktopParty from 'components/DesktopParty'

describe('DesktopParty', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<DesktopParty />)
  })

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
