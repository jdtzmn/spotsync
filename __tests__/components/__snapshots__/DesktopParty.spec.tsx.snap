// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DesktopParty should render correctly 1`] = `<div />`;
